### PR TITLE
chore: update rust toolchain

### PR DESCRIPTION
**Pull request by dpc**

### CODE REVIEW

**Comment by Kodylow** [comment]
how's this only changing flake lock not the flake?

**Comment by dpc** [comment]
> how's this only changing flake lock not the flake?

`fenix` input is not pinned to `rev`, so I just `nix flake lock --update-input fenix` and that updates the pinned version in the lock file, job done.

**Review by bradleystachurski — DISMISSED** [review]
Looks like clippy is complaining after the update, but otherwise looks good :+1:

**Review by bradleystachurski — APPROVED** [review]

File: fedimint-core/src/db/mod.rs [hunk 1]
```diff
@@ -2215,6 +2215,9 @@ pub async fn apply_migrations_server(
     apply_migrations(db, kind, migrations, None, None).await
 }
 
+/// `apply_migrations` iterates from the on disk database version for the
```
Comment (dpc) [inline fedimint-core/src/db/mod.rs:2218]: Clippy now wants a nice and short title on everything.

**Pull request merged**